lib/managers/files.js

```diff
@@ -575,6 +575,73 @@ Files.prototype.deletePermanently = function(fileID, callback) {
 	this.client.del(apiPath, null, this.client.defaultResponseHandler(callback));
 };
 
+
+ /**
+  * Enum of valid actions for locking and unlocking files
+  *
+  * @enum {LockType}
```

Comment:
The enum JSDoc documents the type of the values in the enum, so this would be `@enum {string}`